template.yaml

```diff
@@ -327,6 +341,23 @@ Resources:
         - Key: deregistration_delay.timeout_seconds
           Value: 60
 
+    LoadBalancerListenerTargetGroupECSGreen:
+    Type: 'AWS::ElasticLoadBalancingV2::TargetGroup'
+    Properties:
+      HealthCheckEnabled: TRUE
+      HealthCheckProtocol: HTTP
+      Matcher:
+        HttpCode: 200-499
+      Port: 80
+      Protocol: HTTP
+      TargetType: ip
+      VpcId:
+        Fn::ImportValue:
+          !Sub "${VpcStackName}-VpcId"
+      TargetGroupAttributes:
+        - Key: deregistration_delay.timeout_seconds
+          Value: 5
```

Comment:
Is this set to 5 for a specific reason? (the original target group just above has `deregistration_delay.timeout_seconds` set to 60)